client/my-sites/importer/newsletter/subscribers/paid-subscribers/map-plan.tsx

```diff
@@ -31,18 +33,18 @@ type MapPlanProps = {
 	selectedProductId: string;
 };
 
-function displayProduct( product?: Product ) {
-	if ( ! product ) {
-		return 'Select a Newsletter Tier';
-	}
+type ProductInfoProps = {
+	product: Product;
+};
 
+function ProductInfo( { product }: ProductInfoProps ) {
 	return (
-		<div>
-			<strong>{ product.title || '' }</strong>
+		<>
```

Comment:
Simplified by removing `div` from DOM, but we'll need to check this doesn't break visually.